src/resources/payment_method_ext.rs

```diff
@@ -0,0 +1,25 @@
+use crate::config::{Client, Response};
+use crate::ids::{CustomerId, PaymentMethodId};
+use crate::resources::{PaymentMethod};
+use serde_derive::{Deserialize, Serialize};
+
+/// Attach a payment method to a customer
+///
+/// For more details see [https://stripe.com/docs/api/payment_methods/attach](https://stripe.com/docs/api/payment_methods/attach).
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct AttachPaymentMethod {
+    pub customer: CustomerId,
+}
+
+impl PaymentMethod {
+    /// Attach a payment method to a customer
+    ///
+    /// For more details see [https://stripe.com/docs/api/payment_methods/attach](https://stripe.com/docs/api/payment_methods/attach).
+    pub fn attach(
+        client: &Client,
+        payment_method_id: &PaymentMethodId,
+        attach: AttachPaymentMethod,
```

Comment:
The one open question here is if I should accept a `CustomerId` here instead of going through the ceremony of having users create a `AttachPaymentMethod`